src/Form/index.tsx

```diff
@@ -900,6 +914,13 @@ function Form({
       // need to include value === '' so that we can clear out hidden fields
       if (value !== undefined) hiddenFields[fieldKey] = value;
     });
+    // submit feathery reserved hidden fields
+    fieldValues[FEATHERY_PAYMENTS_SELECTIONS] &&
```

Comment:
could we put these into standard if statements? it's a little hard to read in this format